src/test/java/org/apache/commons/net/tftp/TFTPServer.java

```diff
@@ -610,7 +610,7 @@ protected void finalize() throws Throwable {
     }
 
     /**
-     * Gets the current value for maxTimeoutRetries
+     * Get the current value for maxTimeoutRetries
```

Comment:
Don't change the style of the Javadoc please. A getter "Gets...".

Reply:
I pushed that code by accident, and I already reverted it back. In pull #173 [(that part)](https://github.com/apache/commons-net/pull/173/files#diff-8cfe322813dd0689aac3ae713d40911dfba80a16e7bcad4f6a95d1106e4da5baR613) is fixed